### PR TITLE
Tooltip flickering

### PR DESCRIPTION
**Pull request by nnelgxorz**
# What is it?

- [ ] Feature / enhancement
- [ ] Bug
- [ ] Docs / tests

# Description

Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context. List any dependencies that are required for this change.

# Use cases and why

- 1. One use case
- 2. Another use case

# Screenshots/Demo

# Checklist:

- [ ] My code follows the [developer guidelines of this project](https://github.com/qwikifiers/qwik-ui/blob/main/CONTRIBUTING.md)
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] Added new tests to cover the fix / functionality

### CODE REVIEW

File: packages/website/src/routes/daisy/tooltip/index.tsx [hunk 1]
```diff
@@ -1,5 +1,20 @@
 import { component$ } from '@builder.io/qwik';
-import { Tooltip } from '@qwik-ui/theme-daisy';
+import { Tooltip, TooltipProps } from '@qwik-ui/theme-daisy';
+
+export const positions: TooltipProps['position'][] = [
```
Comment (gilf) [inline packages/website/src/routes/daisy/tooltip/index.tsx:4]: Why not to use a TS enum instead?
`export enum positions { 
   TopEnd = 'top-end', 
   Top = 'top',
   ...
}`

File: packages/headless/src/components/tooltip/tooltip.tsx [hunk 2]
```diff
@@ -64,15 +71,30 @@ export const Tooltip = component$(
       })
     );
 
-    useClientEffect$(({ track }) => {
+    useClientEffect$(async ({ track }) => {
+      let cleanup = () => {
+        console.log('clean up signal has not been set');
```
Comment (gilf) [inline packages/headless/src/components/tooltip/tooltip.tsx:76]: There are too many console.log in this file. While you probably need it for debugging, I don't see any value to commit them as part of the component implementation.

File: packages/website/src/routes/daisy/tooltip/index.tsx (hunk 1 above)
Comment (nnelgxorz) [reply packages/website/src/routes/daisy/tooltip/index.tsx]: They are already typed with the FloatingUI Placement enum.

File: packages/headless/src/components/tooltip/tooltip.tsx (hunk 2 above)
Comment (nnelgxorz) [reply packages/headless/src/components/tooltip/tooltip.tsx]: Still a work in progress. All the console logs will be gone before we merge.

**Comment by gioboa** [comment]
I think we can close it, this PR #159 solved the issue.

**Pull request closed without merging**